File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char,int> um;
        if(s.size()!=t.size())
        {
            return false;
        }
        string a=s;
        string b=t;
        sort(a.begin(),a.end());
        sort(b.begin(),b.end());
        return (a==b);
        
    }
};

File: Data Structures & Algorithms/subsets-ii/submission-0.py
class Solution:
    def subsetsWithDup(self, nums: List[int]) -> List[List[int]]:
        nums.sort()
        ans=[]
        subset=[]
        def dfs(index):
            ans.append(subset[:])
            for i in range(index,len(nums)):
                if i>index and nums[i]==nums[i-1]:
                    continue
                subset.append(nums[i])
                dfs(i+1)
                subset.pop()
        dfs(0)
        return ans



        